File: 蒸蒸简单/构成特定和需要添加的最少元素.py
from math import ceil
from typing import List


class Solution:
    def minElements(self, nums: List[int], limit: int, goal: int) -> int:
        sum = 0
        for num in nums:
            sum += num
        sum = abs(sum - goal)
        res = sum / limit
        return ceil(res)

File: 链蒸蒸简单/构成特定和需要添加的最少元素.java
package 链蒸蒸简单;

//java向右取整
public class 构成特定和需要添加的最少元素 {
    public int minElements(int[] nums, int limit, int goal) {
        long sum = 0;
        for (int num : nums) sum += num;
        sum = Math.abs(sum - goal);
        double res = sum / (double)limit;
        return (int) Math.ceil(res);
    }
}
